updated-knowbe4 (#1304)
SumoLogic/sumologic-documentation: docs/search/search-cheat-sheets/index.md

For a collection of customer created search queries and their use cases, see the [Sumo Logic Community Query Library](https://support.sumologic.com/hc/en-us/community/topics/200396738-Query-Library). 

> For a step-by-step videos creating Sumo Logic queries, see the [Micro Lessons](https://www.youtube.com/playlist?list=PLuHsjJUxgM1fRFUzFZuQcZ2GCW-jtiOxa) or the [Tutorials](https://www.youtube.com/watch?v=_5JHkxG7ZMo&list=PLuHsjJUxgM1cmKQk1UjDUbiUcTnfgNwFl) .  

Reviewer: ```suggestion
For step-by-step videos on creating Sumo Logic queries, see our [Micro Lessons](https://www.youtube.com/playlist?list=PLuHsjJUxgM1fRFUzFZuQcZ2GCW-jtiOxa) and [Tutorials](https://www.youtube.com/watch?v=_5JHkxG7ZMo&list=PLuHsjJUxgM1cmKQk1UjDUbiUcTnfgNwFl).  
```
